### PR TITLE
Temporary input

### PR DESCRIPTION
**Pull request by phabh**
Add trace settings in messages sent by bot for information in debug.

### CODE REVIEW

File: src/Take.Blip.Builder/InputExpirationHandler.cs [hunk 1]
```diff
@@ -85,16 +87,25 @@ public Message ValidateMessage(Message message)
                     throw new ArgumentException("Message content 'StateId' must be present", nameof(InputExpiration.StateId));
                 }
 
+                var messageMetaData = new Dictionary<string, string>()
```
Comment (ceifa) [inline src/Take.Blip.Builder/InputExpirationHandler.cs]: messageMetadata*

File: src/Take.Blip.Builder/InputExpirationHandler.cs [hunk 2]
```diff
@@ -85,16 +87,25 @@ public Message ValidateMessage(Message message)
...
+                    {
+                        { STATE_ID, inputExpiration.StateId },
+                        { IDENTITY, inputExpiration.Identity },
+                    };
+
+                TraceSettings traceSettings = GetTraceSettings(message);
+
+                if( traceSettings != null )
```
Comment (ceifa) [inline src/Take.Blip.Builder/InputExpirationHandler.cs]: Fix spaces:
`if (traceSettings != null)`

File: src/Take.Blip.Builder/InputExpirationHandler.cs [hunk 3]
```diff
@@ -85,16 +87,25 @@ public Message ValidateMessage(Message message)
...
+                {
+                    messageMetaData.Concat(traceSettings.GetDictionary());
```
Comment (andreminelli) [inline src/Take.Blip.Builder/InputExpirationHandler.cs]: This was already merged, but there is a bug here: `Concat` returns a new instance with data concatened, so `messageMetaData` here is not being changed at all....

Comment (andreminelli) [reply src/Take.Blip.Builder/InputExpirationHandler.cs]: Just saw the other PR handling this... Sorry...

**Pull request merged**